util/string.h

```diff
@@ -12,13 +12,32 @@
 #include <cctype>
 #include <string>
 #include <string_view>
+#include <utility>
+#include <vector>
 
 namespace util {
 
 constexpr bool no_case_compare(std::string_view a, std::string_view b) {
     return ranges::equal(a, b, [](auto c1, auto c2) { return std::tolower(c1) == std::tolower(c2); });
 }
 
+inline std::vector<std::string_view> split(std::string_view str, std::string_view sep) {
```

Comment:
I **think** these can be constexpr!

Reply:
Seems like constexpr std::vector is only supported by MSVC at the moment. There is support in trunk builds of GCC and Clang, so I guess we'll have to wait :)